applications/app_balance.c

```diff
@@ -64,6 +64,11 @@ typedef enum {
 	ON
 } SwitchState;
 
+#ifndef BUZZER_ON
```

Comment:
Should this be named differently than the function name, maybe `FOOTSWITCH_BUZZER_ENABLE` or something?